tools/1to2.js

```diff
@@ -0,0 +1,107 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2015 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
+
+var events = require('events'),
+    emitter = new events.EventEmitter(),
+    fs = require('fs'),
+    glob = require('glob'),
+    i, length, removedregex = [];
+
+const removed = [
+  'GetIndexedPropertiesExternalArrayDataLength',
+  'GetIndexedPropertiesExternalArrayData',
+  'GetIndexedPropertiesExternalArrayDataType',
+  'GetIndexedPropertiesPixelData',
+  'GetIndexedPropertiesPixelDataLength',
+  'HasIndexedPropertiesInExternalArrayData',
+  'HasIndexedPropertiesInPixelData',
+  'SetIndexedPropertiesToExternalArrayData',
+  'SetIndexedPropertiesToPixelData'];
+
+length = removed.length;
+
+for (i = 2; i < process.argv.length; i++) {
+  glob(process.argv[i], function (err, matches) {
+    if (err) {
+      throw err;
+    }
+    matches.forEach(function (currentValue) {
+      emitter.emit('file', currentValue);
+    });
+  });
+}
+
+for (i = 0; i < length; i++) {
+  removedregex[i] = new RegExp('^(.*)\\W' + removed[i] + '\\W(.*)', 'm');
+}
+
+function replace(file, s) {
+  var i;
+
+  s = s.replace(/(\W)args(\W)/g, function (match, p1, p2) {
+    return [p1, 'info', p2].join('');
+  });
+
+  s = s.replace('NanScope()', 'NanScope scope');
```

Comment:
All of the string replacements would be better of as regular expressions, with whitespace hardening and whatnot.